manager/Dockerfile

```diff
@@ -44,6 +57,36 @@ RUN python3 -m venv /opt/venv && \
 RUN python3 -m venv /opt/venv && \
         /opt/venv/bin/pip3 --no-cache install -qUr requirements.txt
 
+# Install dependencies
+RUN curl -fL -o /tmp/nmap.tar.bz2 \
```

Comment:
Can you elaborate a bit on why the version of nmap from apt didn't work? It would be easier to maintain this image (and quicker to build) if we don't have to compile nmap.

Reply:
for sure it was best and quickest to use a pre-build nmap, unfortunately tracing the bugs it seemed nmap was responsible for crashing Django.
Celery PRECONDITION_FAILED 1800000 was first noted when the Celery workers stopped. Root cause was this error found in manager logs with nmap v7.8 : 
Target.cc:503: void Target::stopTimeOutClock(const timeval*): Assertion `htn.toclock_running == true' failed
This bug was noted in nmap and fixed in v7.9 (https://www.mail-archive.com/ubuntu-bugs@lists.ubuntu.com/msg5866084.html)

Too bad no apt version is available yet for this version for Bullseye and building 7.9.3 fails if it is not patched: 
(https://github.com/nmap/nmap/issues/2516) so this is the patch in manager/Dockerfile  on line 69-70

another option is to install nmap with snap.

Reply:
Snap is not an option when running in Docker. (like running a container in a container)

Triggered by your comment I'm currently running "FROM ubuntu:22.04" and added nmap (7.9)  with apt in the manager/Dockerfile. Switching from Debian 11 to Ubuntu 22.04 LTS solves the need for manual building nmap. The first tests on this solutions seems working fine. I will test it on big exercises with thousands of nodes.